mongomock/store.py

```diff
@@ -104,3 +107,51 @@ def __len__(self):
     def documents(self):
         for doc in six.itervalues(self._documents):
             yield doc
+
+    def remove_expired_documents(self):
+        for index in six.itervalues(self.indexes):
+            if index.get('expireAfterSeconds') is not None:
+                self._expire_documents(index)
+
+    def _expire_documents(self, index):
+        # Ignore non-integer values
+        try:
+            expiry = int(index['expireAfterSeconds'])
+        except ValueError:
+            return
+
+        # Ignore commpound keys
+        if len(index['key']) > 1:
+            return
+
+        # "key" structure = list of (field name, direction) tuples
+        ttl_field_name = index['key'][0][0]
+        expired_ids = {
```

Comment:
I'm not sure doc['_id'] have to be hashable, so I suggest you build a list instead of building a set.